### ISSUE TITLE
fedora 26 xfce spin

### ISSUE THREAD

**Issue by rdica** [kirsle/.dotfiles#1]
When user runs dnf check-update, it returns 0 packages even when there are pkgs to update.
I had to visudo and add a line for the user to passwdless sudo /usr/bin/dnf check-update.

**Comment by kirsle** [comment]
I couldn't reproduce this.

```
[kirsle@BLUEPC-19 ~]% uname -a 
Linux BLUEPC-19 4.12.8-300.fc26.x86_64 #1 SMP Thu Aug 17 15:30:20 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
[kirsle@BLUEPC-19 ~]% dnf check-update
Last metadata expiration check: 0:00:00 ago on Wed 30 Aug 2017 10:49:00 AM PDT.

augeas-libs.x86_64                                                                                             1.8.1-1.fc26                                                                                             updates               
clang-libs.x86_64                                                                                              4.0.1-4.fc26                                                                                             updates               
...
[kirsle@BLUEPC-19 ~]% echo $?
100
```

**Comment by rdica** [comment]
did you upgrade from 25?

**Comment by kirsle** [comment]
On this system I did. So just now, I checked against the Fedora 26 Live Xfce image in a virtual machine: [screenshot.](https://i.imgur.com/3Sw8y0J.png)

In the left terminal I ran `dnf check-update` and `echo $?`, in the right terminal I ran `pyupdatesd --debug`, and it worked as expected.

By the way, I've created a repo dedicated to this script now: [update-applet](https://github.com/kirsle/update-applet), so it can get more visibility and contributors.

**Comment by rdica** [comment]
Looking over your git repo...I don't have yumex on my install of xfedora
26, I see /usr/bin/yumex-dnf and /usr/bin/yumex-dnf-updatechecker, and
they're part of yumex-dnf.noarch pkg. There is no other yumex pkg:

[root@thinkpad ~]# dnf clean all
81 files removed
[root@thinkpad ~]# dnf search yumex
Fedora 26 - x86_64 - Updates
      801 kB/s |  11 MB     00:14
Fedora 26 - x86_64
      890 kB/s |  53 MB     01:01
RPM Fusion for Fedora 26 - Free - Updates
     525 kB/s | 207 kB     00:00
RPM Fusion for Fedora 26 - Free
     450 kB/s | 519 kB     00:01
RPM Fusion for Fedora 26 - Nonfree - Updates
       38 kB/s | 5.2 kB     00:00
RPM Fusion for Fedora 26 - Nonfree
      386 kB/s | 158 kB     00:00
Last metadata expiration check: 0:00:00 ago on Wed 30 Aug 2017 04:06:50 PM
EDT.
============================================== Name Matched: yumex
==============================================
yumex-dnf.noarch : Yum Extender graphical package management tool



On Wed, Aug 30, 2017 at 2:45 PM, Noah Petherbridge <notifications@github.com

**Comment by kirsle** [comment]
You're right, it should've been `yumex-dnf`. I originally write this script before the yum -> dnf switchover, and personally I never actually click on the update icon, it just reminds me to open a terminal and run a `dnf update` manually... so I didn't notice the command was changed. :smile:
